### DISCUSSION TITLE
SSL on subdomain that is pointing to a third party via CNAME

### DISCUSSION THREAD

**Discussion by iamalismith** [vercel/community#394] [Help] [answered]
I'm trying to set up a third party help center on a subdomain of a vercel site, eg https://help.example.com. 

I've set the relevant CNAME record which points the subdomain at the right place:

name: help
type: CNAME
value: custom.thirdpartyhelpcenter.com

This works, and you can access the helpcenter on that URL, but there are SSL issues as the browser can't verify the url with the certificate which is coming from the helpcenter  and not my site (I think). 

There is a similar question here https://github.com/vercel/community/discussions/34 but the answer given only deals with forwarding a subdomain to a vercel project, rather than this case, which is SSL for a subdomain via a CNAME pointing elsewhere. 😅 

All the info I can find seems to say "SSL works automatically on Vercel" which does seem true for most cases. Any help or pointers would be greatly appreciated! 🙏

**Comment by swarnava** [comment, marked as answer]
Hi,

 Vercel only provides SSL for deployment running on our platform. Since custom.thirdpartyhelpcenter.com points to helpdesk's servers, it is their responsibility to support the SSL. Please reach out to the helpcenter support for more details about how the SSL can be configured with your custom domain.
 
 Thanks!

**Reply by iamalismith** [reply]
Thanks for your explanation, I'll follow up with them 👍
